src/ui/components/SearchForm/component.ts

```diff
@@ -18,14 +17,9 @@ export default class SearchForm extends Component {
 
   submitSearch(event) {
     event.preventDefault();
-    let { search } = this;
     this.args.onSearchByTerm(this.search);
   }
 
-  searchByLocation(event) {
```

Comment:
Is this not necessary anymore?

Reply:
Searching by location is its [own component](https://github.com/simplabs/breethe-client/blob/nojs-search/src/ui/components/SearchCoordinates/component.ts#L8) now.

Reply:
Ah, I see - so this was previously unused anyway…